docs/resource_server.md

```diff
@@ -18,6 +20,37 @@ OIDC_RS_ALLOWED_AUDIENCES=
 
 It implements the resource server using `django-lasuite`, see the [documentation](https://github.com/suitenumerique/django-lasuite/blob/main/documentation/how-to-use-oidc-resource-server-backend.md)
 
+## Customise allowed routes
+
+Configure the `EXTERNAL_API` setting to control which routes and actions are available in the external API. Set it via the `EXTERNAL_API` environment variable (as JSON) or in Django settings.
+
+Default configuration:
+
+```python
+EXTERNAL_API = {
+    "items": {
+        "enabled": True,
+        "actions": ["list", "retrieve", "children", "upload_ended"],
```

Comment:
Seeing `children` makes me think we can't choose between listing children and creating a new one. This is out of the scope of this PR but we should find a way to split this method.

Reply:
Agreed +1